### PR TITLE
jf: add page for JFrog CLI

### PR DESCRIPTION
**Pull request by skofgar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
```
> jf --version
jf version 2.41.0
```

### CODE REVIEW

File: pages/common/jf.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# jf
+
+> Use JF CLI for interfacing with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```
Comment (kbdharun) [inline pages/common/jf.md:3]:
```suggestion
> Interface with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```
How about this? (As we are currently moving from the usage of `CLI` term in page descriptions)

File: pages/common/jf.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Add new configuration:
+
+`jf config add`
+
+- Show current configuration:
+
+`jf config show`
+
+- Artifactory: Search artifacts within given repository and directory:
```
Comment (kbdharun) [inline pages/common/jf.md:14]:
```suggestion
- Search artifacts within the given repository and directory:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/jf.md (hunk 1 above)
Comment (skofgar) [reply pages/common/jf.md:3]: oh that's a good point. I like your suggestion

File: pages/common/jf.md (hunk 2 above)
Comment (skofgar) [reply pages/common/jf.md]: thank you for the suggestions

**Comment by skofgar** [comment]
Thank you for your suggestions and the review @kbdharun, I just started using the JF CLI and will create new PR's as I learn more.

**Review by kbdharun — APPROVED** [review]

File: pages/common/jf.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# jf
+
+> Interface with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
+> More information: <https://jfrog.com/help/r/jfrog-cli/usage>.
+
+- Add new configuration:
```
Comment (acuteenvy) [inline pages/common/jf.md:6]:
```suggestion
- Add a new configuration:
```

File: pages/common/jf.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`jf config add`
+
+- Show current configuration:
```
Comment (acuteenvy) [inline pages/common/jf.md:10]:
```suggestion
- Show the current configuration:
```

File: pages/common/jf.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`jf config show`
+
+- Search artifacts within the given repository and directory:
```
Comment (acuteenvy) [inline pages/common/jf.md:14]:
```suggestion
- Search for artifacts within the given repository and directory:
```

File: pages/common/jf.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# jf
+
+> Interface with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```
Comment (acuteenvy) [inline pages/common/jf.md:3]:
```suggestion
> Interact with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**